### PR TITLE
OpenGL compatibility via metaclass: TracedPath, ParametricFunction, Brace, VGroup

### PR DESCRIPTION
**Pull request by TRoboto**
## Changelog / Overview

Increased OpenGL compatibility using the metaclass mechanism from #1272. `TracedPath`, `ParametricFunction`, `Brace`,  and `VGroup` mobjects are now compatible with both renderers

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Testing Status

All changes were tested locally. Everything seems to work as it should.

## Further Comments

PR #1324 can be closed as I don't see a need for it now.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Review by behackl — APPROVED** [review]
The changes look good to me, thanks for taking care of it! I just want to do some local testing, and additionally make sure that the pipeline failure isn't caused by your changes.

**Comment by TRoboto** [comment]
This PR should be good now.

File: manim/mobject/svg/brace.py [hunk 1]
```diff
@@ -138,10 +145,12 @@ def __init__(
     ):
         self.label_constructor = label_constructor
         self.label_scale = label_scale
-        VMobject.__init__(self, **kwargs)
+        super().__init__(**kwargs)
+
+        self.basecls = OpenGLVMobject if config["renderer"] == "opengl" else VMobject
```
Comment (behackl) [inline manim/mobject/svg/brace.py:150]: I'd prefer not adding `basecls` as an attribute. Does anything speak against replacing `VMobject` by the corresponding group, i.e., `self.get_group_class()`?

**Review by behackl — APPROVED** [review]
I just have a minor comment that I'd like to discuss, but otherwise this looks good to me. (Only read the code, did not test it locally.)

File: manim/mobject/svg/brace.py (hunk 1 above)
Comment (TRoboto) [reply manim/mobject/svg/brace.py]: Nope! I just didn't think of using it.

File: manim/mobject/svg/brace.py [hunk 2]
```diff
@@ -119,14 +123,17 @@ def get_tex(self, *tex, **kwargs):
 
     def get_tip(self):
         # Returns the position of the seventh point in the path, which is the tip.
-        return self.points[28]  # = 7*4
+        if config["renderer"] == "opengl":
+            return self.get_points()[34]
```
Comment (behackl) [inline manim/mobject/svg/brace.py:127]: A comment for where 34 comes from would be great.

**Review by behackl — COMMENTED** [review]
One additional quick question. 👀

File: manim/mobject/svg/brace.py (hunk 2 above)
Comment (TRoboto) [reply manim/mobject/svg/brace.py:127]: Actually I have no idea about this. I had to manually test the number that returns the center of the tip, and 34 is that number.

Comment (behackl) [reply manim/mobject/svg/brace.py:127]: I've opened an issue for this.

**Pull request merged**